priv/signs.json

```diff
@@ -5114,6 +5114,98 @@
       ]
     ]
   },
+  {
+    "id": "babcock_st_eastbound",
+    "headway_group": "green_b",
```

Comment:
Are these the only B branch signs right now?

Reply:
Yep. At least, they are the only signs on "just" the B branch, as opposed to parts shared with other branches.